Sources/MapboxNavigation/NavigationOptions.swift

```diff
@@ -40,6 +40,14 @@ open class NavigationOptions: NavigationCustomizable {
      */
     open var bottomBanner: ContainerViewController?
     
+    /**
+     Configuration for the Predictive Caching.
+     
+     These parameters control how navigation engine will try to predict and pre-load data related to the route.
+     `nil` value means disabling the feature.
```

Comment:
Should the default be to not predictively cache, or should the feature be opt-out?

Reply:
As I can see from Android implementation - it is up to user to [explicitly enable it](https://github.com/mapbox/mapbox-navigation-android-internal/pull/33/files#diff-c88da1a719e50faaaab70f7dc284ae633128af9325f60a0b7355087bb19e6515R163-R166).